recipes/simfil/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.15)
+project(PackageTest CXX)
+
+find_package(simfil CONFIG REQUIRED)
+
+add_executable(test_package src/example.cpp)
+target_link_libraries(test_package simfil::simfil)
```

```suggestion
target_link_libraries(test_package simfil::simfil)
target_compile_features(test_package PRIVATE cxx_std_20)
```

Because Conan 1.x does not specify settings.compiler.cppstd by default